### DISCUSSION TITLE
Standalone mode

### DISCUSSION THREAD

**Discussion by kharar** [tinymovr/Tinymovr#331] [Q&A - Tinymovr] [answered]
My use case is pretty simple, I just need my gimbal motor to run at a fixed speed at boot.
I have Tinymovr M5.2 and a ESP32 setup to communicate over UART with the purpose of setting up this, but when I look at the documentation I can not find any information on how to set motor type=gimbal or enable controller set_velocity mode.
What have I missed?

**Comment by yconst** [comment, marked as answer]
You should not use Gimbal mode with the M5.2, as it is perfectly capable of doing closed loop current control with Gimbal-style motors. 

To work with UART you would typically calibrate, (.Q), wait for it to finish, save config (.S), and then set closed loop mode (.A) and set the velocity control setpoint (e.g. .V20000). This will set velocity control mode implicitly.

Please take a look here for the UART command reference: https://tinymovr.readthedocs.io/en/latest/interfaces/interfaces.html#command-reference

**Reply by kharar** [reply]
Well, the documentation states not to use gimbal mode so that is a bit confusing..
https://tinymovr.readthedocs.io/en/latest/hardware/gimbal.html

**Reply by yconst** [reply]
Exactly, you should not use Gimbal mode with the M5.2

**Reply by kharar** [reply]
Also the Command Reference does not state how to initiate velocity mode.. Is that implicit of sending the .Vxxxx command?

**Reply by yconst** [reply]
I updated the answer above with the step required for closed loop control, please take a look

**Reply by kharar** [reply]
Oh, okay I will try that.

Lastly (hopes) the gimbal page states:

> DO NOT perform calibration on a gimbal motor without setting gimbal mode first!

How do I handle the mode change prior to and after calibration?

**Reply by yconst** [reply]
This refers to the R series as well, Gimbal mode is irrelevant for the M series

**Reply by kharar** [reply]
Thanks a lot for the really fast replies!
